src/Hero6.UserInterface.SierraVga/Entities/Padding.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LateStartStudio.Hero6.UserInterface.SierraVga.Entities
+{
+    public struct Padding
+    {
+        public float Left { get; set; }
```

Comment:
What's the reasoning for this other than meeting the StyleCop standard? It seems to me that each class should be ordered by instantiation during run time.  While that would still place instance-based properties after constructors, that would not include fields (in fact, I believe the StyleCop standard recommends fields before constructors).  However, dividing fields from properties by placing constructors between them seems a little unreadable (I'm referring to properties not backed by fields only).  Personally, I'd prefer fields and simple properties be placed before constructors.  
That being said, I'll switch these over to fields